### ISSUE TITLE
Numeric mask causing unexpected multiplication in Chrome

### ISSUE THREAD

**Issue by chriswoods88** [RobinHerbots/Inputmask#2186]
Applying a numeric mask to an input with an existing value of .50 causes the value to change to 50.00

 - Google Chrome
 - inputmask v4.0.8

**Additional context**

```
{
    alias: "numeric",
    placeholder: '0',
    groupSeparator: ',',
    autoGroup: true,
    allowMinus: true,
    digits: 2,
    digitsOptional: false,
    autoUnmask: true,
    clearMaskOnLostFocus: false,
    positionCaretOnTab: false,
    positionCaretOnClick: "select"
}
```
This mask is getting applied to a number of inputs in my app, everything seems to work just fine, except for when a value comes in without a leading 0 before the decimal point. So if I mask 0.50 everything is great. If I mask .50 I am getting 50.00 in the input. This appears to work just fine in other browsers, specifically tested Edge but don't have Firefox handy right now.

**Comment by RobinHerbots** [comment]
@chriswoods88 , 

Have a try with the 5.x beta branch

**Comment by chriswoods88** [comment]
Thanks Robin. In this case, I'm not able to utilize a beta version in my case, but I was able to fix my issue another way. I'll describe here in case someone else comes across this.

Initially, the reason that the input was getting populated with ".50" is because my C# backend was sending a viewmodel down that included a string "HoursText". I updated that viewmodel to send a decimal instead which causes the input to get populated with "0.50" and the inputmask handles that happily.
